include/boost/graph/named_function_params.hpp

```diff
@@ -364,7 +367,12 @@ BOOST_BGL_DECLARE_NAMED_PARAMS
     template <typename T>
     struct convert_bgl_params_to_boost_parameter {
       typedef typename convert_one_keyword<typename T::tag_type>::type new_kw;
-      typedef boost::parameter::aux::tagged_argument<new_kw, const typename T::value_type> tagged_arg_type;
+      typedef boost::parameter::aux::tagged_argument<
+          new_kw
+        , typename boost::add_const<
+              typename boost::remove_reference<typename T::value_type>::type
+          >::type
+      > tagged_arg_type;
```

Comment:
Why do we add const and remove references from `value_type` now?

Reply:
I had compiler errors before without those type transformations.  I'm testing my local fork now to see if those errors still exist without them and, if so, to report what they are.

Reply:
It looks like I can get by without these type transformations.  I'm going to test my local fork with msvc 14.0 so I can document the regressions.

Reply:
OK, keep us posted.

Reply:
These and other unnecessary changes have now been reverted.